messaginginapp/src/main/java/io/customer/messaginginapp/extension/GistExtensions.kt

```diff
@@ -0,0 +1,22 @@
+package io.customer.messaginginapp.extension
+
+import build.gist.data.model.Message
+import io.customer.messaginginapp.type.InAppMessage
+import io.customer.sdk.extensions.random
+
+fun Message.getNewFromInApp(
+    message: InAppMessage
+): Message = Message(
+    instanceId = message.instanceId,
+    messageId = message.messageId,
+    properties = mapOf(
+        Pair(
+            "gist",
+            mapOf(
+                Pair("campaignId", message.deliveryId)
+            )
+        )
+    )
+)
+
+fun Message.getNewRandom(): Message = getNewFromInApp(InAppMessage(String.random, String.random, String.random))
```

Comment:
Okay so i think this is for testing purposes, shouldn't it be part of `test` directory rather than `main`?

Reply:
Yeah, let me try to move it to `test`.